static/src/javascripts/projects/common/modules/audio/Slider.js

```diff
@@ -2,7 +2,7 @@
 
 import { React, styled } from '@guardian/dotcom-rendering/packages/guui';
 
-const Scrubber = styled('div')(({ left }) => ({
+const Scrubber = styled('input')(({ left }) => ({
```

Comment:
I'm surprised there's no impact on rendering here.

Reply:
Yeah.. can revert this, I thought it would help with the bug but think it's not needed now